tests/quantization/bnb/test_mixed_int8.py

```diff
@@ -138,7 +138,9 @@ def test_get_keys_to_not_convert(self):
             model_id, trust_remote_code=True, revision="72e5f594ce36f9cabfa2a9fd8f58b491eb467ee7"
         )
         with init_empty_weights():
-            model = AutoModelForCausalLM.from_config(config, trust_remote_code=True)
+            model = AutoModelForCausalLM.from_config(
+                config, trust_remote_code=True, revision="72e5f594ce36f9cabfa2a9fd8f58b491eb467ee7"
```

Comment:
```suggestion
                config, trust_remote_code=True, code_revision="72e5f594ce36f9cabfa2a9fd8f58b491eb467ee7"
```
Here I could also pass `code_revision` but makes the API a bit inconsistent (different behaviour between `revision` and `code_revision` no?